### PR TITLE
Bump version, add development instructions...

### PR DESCRIPTION
**Pull request by MrCreosote**
# Description of PR purpose/changes

Bump version, add development instructions and add an interactive HTML version of the spec.

# Jira Ticket / Github Issue #
- [n/a] Added the Jira Ticket to the title of the PR e.g. (DATAUP-69 Adds a PR template)

# Testing Instructions
* Details for how to test the PR: 
- [X] Tests pass in Github Actions and locally 
- [X] Changes available by spinning up a local test suite

# Dev Checklist:

- [X] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/development
- [X] I have performed a self-review of my own code
- [n/a] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [?] My changes generate no new warnings - 3k warnings
- [n/a] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [n/a] Any dependent changes have been merged and published in downstream modules
- [X] I have run Black and Flake8 on changed Python Code manually or with git precommit (and the Github Actions build passes)

# Updating Version and Release Notes (if applicable)

- [X] [Version has been bumped](https://semver.org/) for each release
- [n/a] [Release notes](/RELEASE_NOTES.md) have been updated for each release (and during the merge of feature branches)

### CODE REVIEW

File: KIDLspec.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+html, body {
```
Comment (MrCreosote) [inline KIDLspec.css:1]: SDK generated code

File: README.md [hunk 2]
```diff
@@ -80,6 +80,30 @@ pre-commit uninstall
...
+* To create a new release:
+    * Increment the version as per [semantic versioning](https://semver.org/) in `kbase.yml`.
+        * Update the release notes to the correct version, if necessary.
+    * Run `make compile`.
+    * Go through the process above to get the changes into `develop`.
+    * Make a PR from `develop` to `main`.
+    * Once the PR is apporoved, merge (no squash) to `main`.
+    * Tag the merge commit in GitHub with the semantic version from `kbase.yml`.
  
 # Help
```
Comment (MrCreosote) [inline README.md:109]: HTML file below is SDK generated code

File: lib/execution_engine2/execution_engine2Impl.py [hunk 3]
```diff
@@ -28,9 +28,9 @@ class execution_engine2:
     # state. A method could easily clobber the state set by another while
     # the latter method is running.
     ######################################### noqa
-    VERSION = "0.0.1"
```
Comment (MrCreosote) [inline lib/execution_engine2/execution_engine2Impl.py:31]: SDK generated code

**Review by ialarmedalien — APPROVED** [review]
👍

**Comment by MrCreosote** [comment]
LGTM has been running for an hour, so it seems like it's having issues currently.

**Pull request merged**